src/main/java/Event.java

```diff
@@ -16,6 +16,6 @@ public String getAt() {
 
     @Override
```

Comment:
Use of inherence improves the layout of the project and as well as the readability